src/main/java/org/mockito/BDDMockito.java

```diff
@@ -304,12 +304,22 @@ public T should(InOrder inOrder, VerificationMode mode) {
         /**
          * @see #verifyZeroInteractions(Object...)
          * @since 2.1.0
+         * @deprecated Please migrate your code to {@link #shouldHaveNoInteractions()}
```

Comment:
Let's add a `@deprecated since 3.0.0` here, so we don't accidently remove it in the wrong version (which we did previously)